css/style.css

```diff
@@ -349,3 +349,12 @@ h4 {
   top: 0;
   width: 100%;
 }
+
+@media print {
```

Comment:
If I recall correctly, this is no different than using the `media='print'` property on the `<link>` tag...

You shouldn't need both this and a separate file.